kll/sqlx/kll_sketch_float_get_rank.sqlx

```diff
@@ -24,12 +24,19 @@ RETURNS FLOAT64
 LANGUAGE js
 OPTIONS (
   library=["gs://$GCS_BUCKET/kll_sketch.js"],
-  description = '''Returns an approximation to the normalized rank of the given value. from 0 to 1, inclusive.
+  description = '''Returns an approximation to the normalized rank, on the interval (0.0, 1.0], of the given value.
```

Comment:
why zero is not included? the rank can be zero

Reply:
Oops.  Copy/paste error.